File: FreeCodeCamp/Javascript/largestNumbers.js
// https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/return-largest-numbers-in-arrays
// Reprak11
function largestOfFour(arr) {
    let numMAX = arr.map(element => Math.max(...element))
    return numMAX;
}

console.log(largestOfFour([[4, 9, 1, 3], [13, 35, 18, 26], [32, 35, 97, 39], [1000000, 1001, 857, 1]]));

File: FreeCodeCamp/Python/largestNumbers.py
# https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/return-largest-numbers-in-arrays
# Reprak11

def largestOfFour(arr):
    numMAX = []
    for i in arr:
        numMAX.append(max(i))
    return numMAX

if __name__ == "__main__":
    print(largestOfFour([[17, 23, 25, 12], [25, 7, 34, 48], [4, -10, 18, 21], [-72, -3, -17, -10]]))